docs/site/migration/auth/example.md

```diff
@@ -0,0 +1,570 @@
+This example is migrated from
+[loopback-example-access-control](https://github.com/strongloop/loopback-example-access-control),
+and uses the authentication and authorization system in LoopBack 4 to implement
+the access control.
+
+# App Scenario
+
+This example is a donation system. Users can view or make donations to projects
```

Comment:
In the original [loopback-example-access control README](https://github.com/strongloop/loopback-example-access-control#loopback-example-access-control), it has a pretty good description of what the scenario is. Do you want to reuse that?

Reply:
reused it